Library/Homebrew/test/formula_spec.rb

```diff
@@ -699,6 +699,27 @@
     end
   end
 
+  specify "#service" do
+    f = formula do
+      url "https://brew.sh/test-1.0.tbz"
+      service do
+        arguments ["/usr/local/bin/beanstalkd"]
+        error_path "/var/log/beanstalkd.error.log"
+        log_path "/var/log/beanstalkd.log"
+        working_dir "/var"
+        keep_alive true
+        run_at_load true
```

Comment:
Trying to think how these would apply on Linux.

Reply:
First distinguishes the `restart mode` on linux. Second is now removed